assets/chat/js/chat.js

```diff
@@ -415,7 +415,13 @@ class Chat {
 
     // ESC
     document.addEventListener('keydown', (e) => {
-      if (isKeyCode(e, KEYCODES.ESC)) ChatMenu.closeMenus(this); // ESC key
+      if (isKeyCode(e, KEYCODES.ESC)) {
+        const activeView = this.getActiveWindow().scrollplugin;
+        // If any menus are open, close them first
+        if (this.getActiveMenu() !== undefined) ChatMenu.closeMenus(this);
```

Comment:
You can just do `this.getActiveMenu()` in the if statement without the comparison; `undefined` is falsy and the menu object is truthy, so it ends up working anyway.

Reply:
Updated (‾◡◝)